### PR TITLE
refactor(FOROME-905): do redesign of drawer note modal

### PR DESCRIPTION
**Pull request by QYaroslavTrefilov**
https://quantori.atlassian.net/browse/FOROME-905

### CODE REVIEW

File: src/i18n/locales/en.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ export const en = {
     showSelectionOnly: 'Show selection only',
     saveNote: 'Save note',
     noDataToShow: 'No data to show',
+    textAboutSomething: 'Text about something',
```
Comment (AndreyUstyumenko) [inline src/i18n/locales/en.ts:149]: I don't think it's a valid placeholder

File: src/components/variant/ui/drawer-note/drawer-note.tsx [hunk 2]
```diff
@@ -74,33 +107,46 @@ const DrawerNoteModal = observer(({ close }: any) => {
...
         <textarea
-          placeholder="Enter text"
-          className="w-full mt-2 p-3 h-80 rounded-lg resize-none mx-auto"
-          rows={15}
+          ref={textareaRef}
+          placeholder={t('variant.textAboutSomething')}
           value={value}
-          onChange={(e: ChangeEvent<HTMLTextAreaElement>) =>
-            handleChange(e.target.value)
+          onChange={(event: ChangeEvent<HTMLTextAreaElement>) =>
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/drawer-note/drawer-note.tsx]: better pass a function without creating anonymous one and grab the value inside of it

File: src/components/variant/ui/drawer-note/drawer-note.tsx [hunk 3]
```diff
@@ -65,6 +68,36 @@ const DrawerNoteModal = observer(({ close }: any) => {
...
+    // Calculate the height
+    const height =
+      parseInt(computed.getPropertyValue('border-top-width'), 10) +
+      parseInt(computed.getPropertyValue('padding-top'), 10) +
+      textareaRef.current.scrollHeight +
+      parseInt(computed.getPropertyValue('padding-bottom'), 10) +
+      parseInt(computed.getPropertyValue('border-bottom-width'), 10)
+
+    textareaRef.current.style.height = `${height}px`
+  }, [textAreaValue])
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/drawer-note/drawer-note.tsx]: maybe move to a custom hook?

**Review by Lapk1n — APPROVED** [review]

File: src/components/variant/ui/drawer-note/drawer-note.tsx (hunk 2 above)
Comment (QYaroslavTrefilov) [reply src/components/variant/ui/drawer-note/drawer-note.tsx]: Fixed it

File: src/components/variant/ui/drawer-note/drawer-note.tsx (hunk 3 above)
Comment (QYaroslavTrefilov) [reply src/components/variant/ui/drawer-note/drawer-note.tsx]: Made it

File: src/i18n/locales/en.ts (hunk 1 above)
Comment (QYaroslavTrefilov) [reply src/i18n/locales/en.ts:149]: I took a placeholder from figma, what is better to use instead?

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**